tests/integration/modules/test_beacons.py

```diff
@@ -44,15 +44,15 @@ def test_add_and_delete(self):
             ['ps', [{'processes': {'apache2': 'stopped'}}]],
             f_timeout=300
         )
-        self.assertTrue(_add['result'])
+        self.assertTrue(_add if isinstance(_add, bool) else _add['result'], _add)
```

Comment:
This shouldn't be needed since we resolved the test failure. Also, if `_add` is not a dictionary something went wrong. I think it makes more sense to use `assertSaltTrueReturn` rather than doing the inline if else.